tests/unit/list/dlist.c

```diff
@@ -327,6 +327,90 @@ void test_dlist(void)
 	zassert_equal(ii, 2, "");
 }
 
+int cond(sys_dnode_t *node, void *data)
+{
+	return (node == data) ? 1 : 0;
+}
+/**
+ * @brief Verify doubly linked list funtionalities
+ *
+ * @see sys_dlist_is_head(),sys_dlist_is_tail(),
+ * sys_dlist_has_multiple_nodes(),sys_dlist_get()
+ * sys_dlist_peek_head_not_empty(),sys_dlist_insert_at(),
+ * sys_dlist_peek_prev(),
+ */
+void test_dlist2(void)
+{
+	struct container_node test_node[6];
+	struct container_node insert_node;
+	struct container_node insert_node2;
+
+	/* Initialize */
+	memset(test_node, 0, sizeof(test_node));
+	memset(&insert_node, 0, sizeof(insert_node));
+	memset(&insert_node2, 0, sizeof(insert_node2));
+	sys_dlist_init(&test_list);
+
+	/* Check if the dlist is empty */
+	zassert_true(sys_dlist_get(&test_list) == NULL,
+			"Get a empty dilst, NULL will be returned");
+
+	/* Check if insert node inta a empty dlist */
```

Comment:
It's better to change it like below

"Check if a node can be inserted into an empty dlist."